src/tests/system/tests/test_logging.py

```diff
@@ -14,147 +17,121 @@
 
 
 @pytest.mark.topology(KnownTopology.Client)
-def test_logging__default_debug_level_check(client: Client):
+def test_logging__default_settings_logs_debug_level(client: Client):
     """
-    :title: Check default debug level when sssd started successfully
+    :title: Default settings writes the debug level to logs
     :setup:
-        1. Clear logs and cache
-        2. Start SSSD with default debug level
+        1. Configure SSSD for local system authentication
+        2. Clear logs and start SSSD with default debug level
     :steps:
         1. Check log files
     :expectedresults:
-        1. "Starting with debug level = 0x0070" is in each file and
-            if log contains more than one line, log message with number "0x3f7c0" is stored
+        1. Logs messages contain default debug level 0x0070
     :customerscenario: False
     """
     client.sssd.common.local()
     client.sssd.default_domain = "local"
 
-    client.sssd.clear(db=True, memcache=True, logs=True, config=False)
+    client.sssd.clear(logs=True)
     client.sssd.start(debug_level=None)
 
     for file in [client.sssd.logs.monitor, client.sssd.logs.domain(), client.sssd.logs.nss, client.sssd.logs.pam]:
         log_str = client.fs.read(file)
-        assert "Starting with debug level = 0x0070" in log_str, f"Log file has wrong format: {log_str}"
+        assert "Starting with debug level = 0x0070" in log_str, "Debug level value are not in logs!"
```

Comment:
I would read actual value and assert 'level == 0x0070'

Reply:
Exit message is still misleading.

If there will be "Starting with debug level = 0x0010" line in the log it is not  "Debug level value are not in logs!"
It is "Found unexpected value 0x0010 instead of 0x0070" as a log level.